engine/src/main/java/org/hibernate/validator/internal/xml/ClassLoadingHelper.java

```diff
@@ -21,7 +21,7 @@
  *
  * @author Gunnar Morling
  */
-/*package*/ class ClassLoadingHelper {
+public class ClassLoadingHelper {
```

Comment:
I haven't my IDE handy but can't you move it to the package now using it? Is it now used by multiple packages?

Reply:
yeah ... wasn't really sure what to do about it. We create this helper in `org.hibernate.validator.internal.xml.MappingXmlParser` and this single instance is passed to almost all builders from `org.hibernate.validator.internal.xml.stax.mapping` where it is actually used.

Reply:
I think I would have:
 * `org.hibernate.validator.internal.xml.mapping` without `stax` for all the mapping classes, including the `MappingXmlParser`
 * the same for `config`

That would solve the problem, wouldn't it?